crates/bitwarden-ssh/src/error.rs

```diff
@@ -9,3 +9,16 @@ pub enum KeyGenerationError {
     #[error("Failed to convert key: {0}")]
     KeyConversionError(ssh_key::Error),
 }
+
+#[bitwarden_error(flat)]
+#[derive(Error, Debug, PartialEq)]
+pub enum SshKeyImportError {
+    #[error("Failed to parse key")]
+    ParsingError,
+    #[error("Password required")]
+    PasswordRequired,
+    #[error("Wrong password")]
+    WrongPassword,
+    #[error("Unsupported key type")]
+    UnsupportedKeyType,
+}
```

Comment:
❓ @Hinton - Is there a path forward for i18n in the SDK? These seem like the kind of errors we'd like to forward to the user, which would mean communicating the i18n key we'd like the UI to use for localization along with the error.

Reply:
No we leave that to the presentation layer for now. Using i18n in lower level services have always been sort of a crutch.

For WASM we support https://sdk-api-docs.bitwarden.com/bitwarden_error_macro/attr.bitwarden_error.html which exposes basic, flat and full. For non user facing errors it's safe to use `basic` or `flat` and log it. For user facing errors we generally encourage you to use `flat` or `full`, and properly handle the error in the presentation layer.

In this case I suspect we want to show error messages to the user and should use `flat` or `full` and display different errors per variant.

Reply:
Thanks! I'm going to leave this item open for future reference, but it's not blocking the review.